src/note.ts

```diff
@@ -11,7 +11,7 @@
  *
  * @property {string[]} stemDirectionNames - an Array of allowable stemDirection names.
  */
-import Vex from 'vexflow';
+import { Accidental, StaveNote, Stem } from 'vexflow';
```

Comment:
Because Music21 has an Accidental class as well, I will import this as VFAccidental.